go/api.go

```diff
@@ -161,13 +161,13 @@ func (page *EventPageSingleType[T]) Event(partitionID int, h map[string]string,
 }
 
 // Handler wraps API in a http.Handler.
-func Handler(logger logrus.FieldLogger, api API) http.Handler {
+func Handler(path string, logger logrus.FieldLogger, api API) http.Handler {
```

Comment:
Hmm. I think the way this is used, it's placed as a sub-handler on another path elsewhere, so you'd just add feed/v1 there and there's no need to take a path there.

Perhaps the whole use of mux could just be removed now and the whole thing simplified a lot more.

@dtitov do you agree?

Reply:
I retract this now -- it's actually a huge advantage that all use of this function breaks, so that's a good enough reason for now to keep this in. Otherwise the code silently drops the suffix when upgrading server version.

So keep it!

Reply:
I'll hold off for a bit until you decide what is the best option here.

Reply:
As you can see I have a vocal argument with myself :)

Let's wait for other opinions. A rename to `HandlerWithoutPath` (without there being a Handler with path...), while not adding the argument, is another way to make a breaking change of course. I think a breaking change would be good here to avoid bugs being deployed.

Reply:
@dtitov  what do you think? I don't know if Kenneth is a native in Go so if we don't merge as-is, I think perhaps you would have to push a change where we don't use `mux` and rename it to `HandlerWithoutPath`.

I'm OK with merging this.

Reply:
(Apologies if you are Kenneth. I really have no clue about it..)